google/ggrc-core
# GGRC-4249 User with changed role act as user with ex-role

# DOES NOT REVIEW THIS
# Issue description
When we reassign role from Role A (for example, WF Admin) to Role B (for example, WF Member) then, if we did not refresh the page, we still have ability to make actions like for Role A (Admin). 

# Steps to test the changes 
1. Create some of this objects: assessment, wf, standard, regulation, requirement, objective, product, systems, metrics, technology env, product groups, proccesses, audit, access group, contract, data asset, document, evidence, facilities, key report, markets, org groups, policies, programs, projects, tasks, threats, vendors. 
2. Delete yourself from role 
**Actual Result**: You have ability to make actions like for role that was deleted. 
**Expected result**: Page should reload with new permisions. 
# Solution description 
Refresh the page after delete yourself on all objects with inline edit (assessment, wf, standard, regulation, requirement, objective, product, systems, metrics, technology env, product groups, proccesses, audit, access group, contract, data asset, document, evidence, facilities, key report, markets, org groups, policies, programs, projects, tasks, threats, vendors).
# Sanity checklist 

# PR Review checklist
